visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +137,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
```

Comment:
> Precisely speaking, for stopping of a trajectory, the service call 
> with |wait_for_execution:=false| and |stop()| method call is enough. 
> The reason I needed to change the |execute()| method to |move()| is to 
> get the trajectory execution server status to handle GUI button status 
> correctly in 5503fa9 
> <https://github.com/ros-planning/moveit_ros/pull/709/commits/5503fa925adbf5a795d4b5c7095117ccb1724f03>.
>
The service call doesn't have a stop() method, does it? I guess you are 
intermixing with the move action?